tests/config/prettier-entry.js

```diff
@@ -0,0 +1,4 @@
+
+import getPrettier from "./get-prettier.js"
+
+export default await getPrettier();
```

Comment:
What should we do about this, Node.js 12 don't support top level `await`

1. Don't run test on Node.js 12
2. Refactor format test to be async, this is going to be hard, currently we run main parser format before `test()`, and Jest don't allow async call in `describe`

Reply:
Never mind, lucky we are running a custom runner, I managed to make it work https://github.com/prettier/prettier/pull/12272/commits/c7409c658c53e0dcf513aeb8956a14e46bd555e1